controllers/toolchaincluster/healthchecker_test.go

```diff
@@ -123,8 +122,8 @@ func withStatus(conditions ...toolchainv1alpha1.ToolchainClusterCondition) toolc
 	}
 }
 
-func newToolchainCluster(name, apiEndpoint string, status toolchainv1alpha1.ToolchainClusterStatus) (*toolchainv1alpha1.ToolchainCluster, *corev1.Secret) {
-	toolchainCluster, secret := test.NewToolchainClusterWithEndpoint(name, "secret", apiEndpoint, status, map[string]string{})
+func newToolchainCluster(name, ons string, apiEndpoint string, status toolchainv1alpha1.ToolchainClusterStatus) (*toolchainv1alpha1.ToolchainCluster, *corev1.Secret) {
+	toolchainCluster, secret := test.NewToolchainClusterWithEndpoint(name, ons, "secret", apiEndpoint, status, map[string]string{"namespace": "test-namespace"})
```

Comment:
this is another place where `ons` should be renamed

Reply:
done